port/platform/platformio/example/esp32-cell-thingstream-mqttsn.cpp

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright 2019-2023 u-blox
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ *
+ * A simple demo application showing how to set up
+ * a MQTT-SN connection to thingstream using the ubxlib.
```

Comment:
Change MQTT-SN connection comment to:

"a connection to the Thingstream MQTT-Anywhere service (MQTT-SN) using the ubxlib."

Reply:
And this also ;-)